staker/staker.go

```diff
@@ -683,6 +683,43 @@ func (s *Staker) Act(ctx context.Context) (*types.Transaction, error) {
 		StakeExists:          rawInfo != nil,
 	}
 
+	if s.config.EnableFastConfirmation {
```

Comment:
as we discussed, we also need a solution for the WatchTower that missed confirming an assertion, or who'se confirm transaction was sent but was later lost

Reply:
Updated to account for this